Feedback

jasminweismann opened the pull request:
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>


Subscribed: @Rashad-Khoshnow

jasminweismann committed:
Setting up GitHub Classroom Feedback

Rashad-Khoshnow committed:
Aufgabe erledigt

gerhofer commented on aufgabe/zweiPunktZwei.java:
```diff
@@ -0,0 +1,25 @@
+
+        System.out.println("number is: " + number);
+
+        if(number % 10 == 0){
```
bitte nochmal die autoformattierung drüber laufen lassen (Strg+Alt+L im IntelliJ)

gerhofer commented on aufgabe/zweiPunktZwei.java:
```diff
@@ -0,0 +1,25 @@
+        if(number % 10 == 0){
+            System.out.println("Volle");
+        }else if (number % 2 == 0) {
+            System.out.println("gerade");
```
Wenn ich eine Runde zahl eingeb ist die ja voll und gerade - das möcht ich auch auf der konsole ausgedrückt bekommen

gerhofer commented on aufgabe/zweiPunktZwei.java:
```diff
@@ -0,0 +1,25 @@
+            System.out.println("That is my lucky number");
+
+            }
+        if (luckyNumber > 9) {
```
aber 100 wär doch dreistellig nicht zweistellig? und ist -10 nicht auch eine zweistellige Zahl 🤔 ?

Rashad-Khoshnow committed:
Runde und gerade zahl funktioniert.
Ab drestellig ist es jetzt auch kein zweistellig mehr und ab -10 gibt das system jetzt auch zweistellig aus.

gerhofer reviewed (changes requested):
Bitte folgendes machen: 
* autoformattierung laufen lassen (strg+alt+L) 

außerdem haben wir noch ein Problem wenn wir z.b. 10 eingeben erwart ich mir, dass mir gesagt wird dass die zahl rund ist und gerade - du machst dir hier das leben schwerer in dem du unabhängige Bedingungen mit `else if` verknüpfst. 

Du würdest dir leichter tun für jede "Abfrage" einfach ein if zu verwenden (if gerade else ungerade, if rund else unrund, ...)

Rashad-Khoshnow committed:
Autoformatiert

gerhofer reviewed (approved):
super 👍
